### PR TITLE
Issue #13345: Enabled IllegalThrows example tests

### PR DESCRIPTION
**Pull request by SteLeo1602**
Issue: #13345

Enabled example tests for IllegalThrows check.

### CODE REVIEW

**Comment by SteLeo1602** [comment]
GitHub, generate website

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegalthrows/Example1.java [hunk 1]
```diff
@@ -5,15 +5,19 @@
...
 </module>
 */
+package com.puppycrawl.tools.checkstyle.checks.coding.illegalthrows;
 
 // xdoc section -- start
-public class Test {
+public class Example1 {
   public void func1() throws RuntimeException {} // violation
-  public void func2() throws Exception {}  // ok
+  public void func2() throws Exception {}
```
Comment (romani) [inline src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegalthrows/Example1.java:13]: Apply to all: `void f2() throws Exception {}`
To make them horizontal short.

**Review by romani — CHANGES_REQUESTED** [review]
All good, but on phone I need to scroll horizontally to see violations.
Please do me a favor:

**Comment by romani** [comment]
GitHub, generate website

**Pull request merged**